packages/world/src/systems/BuildSystem.sol

```diff
@@ -96,11 +96,11 @@ library BuildLib {
 contract BuildSystem is System {
   using ObjectTypeLib for ObjectTypeId;
 
-  function buildWithExtraData(
+  function buildWithDirection(
     ObjectTypeId buildObjectTypeId,
     Vec3 baseCoord,
     Direction direction,
-    bytes memory extraData
+    bytes calldata extraData
```

Comment:
ahh nice I assume using calldata over memory also saves gas?

Reply:
Yeah in some cases, when the function doesn't need to copy it to memory